python/src/LevelSet_doc.i.in

```diff
@@ -75,13 +75,13 @@ True"
 
```

Comment:
Can't suggest it using the GitHub interface, but line 40 should be changed to :

"Return the union between this LevelSet and another one as a LevelSet.